ucp/benchmarks/README.md

```diff
@@ -0,0 +1,117 @@
+# Running Benchmarks
+
+## cuDF Merge
+
+The cuDF merge benchmark can be executed in 3 different ways:
+
+1. Local-node only;
+1. Multi-node with automatic process launch via SSH;
+1. Multi-node with manual process launch.
+
+In the following subsections we will exemplify how to launch the benchmark for each case. Be sure to check `python -m ucp.benchmarks.cudf_merge --help` to see a complete description of available options and their description.
+
+
+### Local-node only
+
+This is the simplest setup and will execute the benchmark only on GPUs where the process is being launched from. In its simple form, it can be executed as:
```

Comment:
```suggestion
This is the simplest setup and will execute the benchmark only using GPUs on the host the process is being launched from. In its simple form, it can be executed as:
```